app/src/main/java/com/example/bigowlapp/activity/SetScheduleActivity.java

```diff
@@ -10,6 +10,7 @@
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+        moveTaskToBack(true);
```

Comment:
This is causing the app to go to background when Set Schedule button is clicked. Please remove it.

Reply:
Yes sure. This is something I had for testing and I needed to remove after that